day-one-challenge/tyranny-of-the-rocket/tests/getFuelRequirements.test.js

```diff
@@ -0,0 +1,17 @@
+const { expect } = require('chai');
+const totalfuel = require('../getFuelRequirements');
+
+describe('totalfuel', () => {
+  it('outputs the correct amount of fuel when given a single input', () => {
+    expect(totalfuel([100756])).to.equal(50346);
+  })
+
+  it('outputs the correct amount of fuel when given multiple inputs', () => {
+    expect(totalfuel([24, 37, 1009, 2056])).to.equal(1512);
+  })
+
+  it('treats negative fuel as zero', () => {
+    const equal = () => { if (totalfuel[0] === totalfuel[-6]) return true; };
+    expect(equal()).to.equal(true);
+  })
```

Comment:
This is interesting. I haven't thought of arranging in this way. I do see what you are saying with this test. However, boolean results are not very friendly when they fail. The test just says, 'expected true to the false' or something. If you instead assert the number `0` i think you'll get better output. Another option is that `expect` can take another argument that is a string that is output when the test fails. Thoughts?

Reply:
Thanks for the heads up, I've refactored the test here: efd23ee . Let me know what you think.